sim/common/shared/shared_utils.go

```diff
@@ -604,7 +604,8 @@ type IgniteConfig struct {
 	DotAuraTag         string
 	ProcTrigger        core.ProcTrigger // Ignores the Handler field and creates a custom one, but uses all others.
 	DamageCalculator   IgniteDamageCalculator
-	IncludeAuraDelay   bool // "munching" and "free roll-over" interactions
+	NumberAOEOfTargets int32 // Number of targets the Ignite tick will hit. defaults to 1.
```

Comment:
Typo, should be `NumberOfAoeTargets`. But I would shorten it to just `NumAoeTargets`.